modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/RexImpTable.java

```diff
@@ -632,7 +632,7 @@ Builder populate() {
       defineMethod(DEGREES, "degrees", NullPolicy.STRICT);
       defineMethod(POW, "power", NullPolicy.STRICT);
       defineMethod(RADIANS, "radians", NullPolicy.STRICT);
-      defineMethod(ROUND, "sround", NullPolicy.STRICT);
+      defineIgniteMethod(ROUND, "sround", NullPolicy.STRICT);
```

Comment:
all ignite's overrides better to place to `populateIgnite()`

Reply:
Fixed.